src/screens/TrackingFlights/TrackingFlights.jsx

```diff
@@ -1,9 +1,71 @@
-import React from 'react';
```

Comment:
I think this component should be names `FavouriteFlights` or something like that.